microsoft/testsuites/core/disk_demo.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+    search_space,
+)
+from lisa.features import Disk
+from lisa.features.disks import DiskStandardHDDLRS
+from lisa.node import Node
+from lisa.schema import DiskType, DiskOptionSettings
+from lisa.tools import Echo, Cat, Mount
+@TestSuiteMetadata(
+    area="first test suite",
+    category="functional",
+    description="""
+    this is an example test suite.
+    it shows a basic sample of writing a test case.
+    it also is a good chance to learn about the testsuites.
+    """,
+)
+class FirstTestSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        Demo writing a test and using the disk.
+        """,
+        priority=4,
+        requirement=simple_requirement(
+            disk=DiskOptionSettings(
+                disk_type=DiskType.StandardHDDLRS,
+                data_disk_count=search_space.IntRange(min=1),
+            )   
+        ),
+    )
+    def check_disk_move_operation_rydailey(self, node: Node, log: Logger,) -> None:
+        # write hello world to a file
+        text = "Hello World!"
+        filepath = "/sample.txt"
+        diskpath = "/disk"
+        node.tools[Echo].write_to_file(text,filepath,sudo=True)
+        # mount the disk if it isn't mounted
+        disk = node.features[Disk]
+        disks = disk.get_raw_data_disks()
+        mount = node.tools[Mount]
+        mount.mount(disks[0],diskpath,format=True)
+        # move the text file to the disk
+        node.execute(
+                f"mv {filepath} {diskpath}",
```

Comment:
We try not to run commands directly and instead create a tool for it. This is more pythonic way and cleaner.

Reply:
Didn't see a mv tool available. Should I look for a different tool or generate a mv tool just for this exercise?

Reply:
You can create a move tool

Reply:
@r-dailey Any updates on this?